components/homme/src/theta-l_kokkos/cxx/cxx_f90_interface_theta.cpp

```diff
@@ -581,17 +581,19 @@ void init_boundary_exchanges_c ()
     bmm[MPI_EXCHANGE_MIN_MAX]->set_connectivity(connectivity);
   }
 
-  if (params.transport_alg == 0) {
-    // Euler BEs
-    auto& esf = c.get<EulerStepFunctor>();
-    esf.reset(params);
-    esf.init_boundary_exchanges();
-  } else {
+  if (params.qsize > 0) {
```

Comment:
Does theta formulation make any sense without qv?

Reply:
Yes, it can run 'dry'.